codegen/java-gen/src/test/scala/com/lightbend/akkasls/codegen/java/ViewServiceSourceGeneratorSuite.scala

```diff
@@ -208,15 +207,11 @@ class ViewServiceSourceGeneratorSuite extends munit.FunSuite {
       generatedSrc,
       """package com.example.service;
        |
-       |import com.akkaserverless.javasdk.impl.view.UpdateHandlerNotFound;
-       |import com.akkaserverless.javasdk.impl.view.ViewRouter;
-       |import com.akkaserverless.javasdk.view.View;
+       |import com.akkaserverless.javasdk.view.ViewProvider;
```

Comment:
Imports are currently all over the place and we should sort them ultimately, but that will require touching all tests.